PWGHF/D2H/Tasks/taskBs.cxx

```diff
@@ -124,7 +124,7 @@ struct HfTaskBs {
     registry.add("hYGenWithProngsInAcceptance", "MC particles (generated-daughters in acceptance);B^{0}_{s} candidate #it{y}^{gen};entries", {HistType::kTH2F, {{100, -2., 2.}, axisPt}});
 
     if (checkDecayTypeMc) {
-      constexpr uint8_t kNBinsDecayTypeMc = DecayTypeMc::NDecayTypeMc;
+      constexpr uint8_t kNBinsDecayTypeMc = DecayTypeMc::OtherDecay;
```

Comment:
This will make `kNBinsDecayTypeMc = 2` so `labels` will only have 2 elements and `labels[DecayTypeMc::OtherDecay] = "Other decays";` will crash.